platform-sdk/consensus-gossip-impl/src/main/java/module-info.java

```diff
@@ -1,5 +1,9 @@
 module com.hedera.consensus.gossip.impl {
+    exports com.hedera.service.gossip.impl;
+
+    requires transitive com.swirlds.common;
```

Comment:
Is this for a new publish group? I didn't think we were publishing anything new to com.swirlds.*